exchanges/devtools/src/declarations/urql.d.ts

```diff
@@ -0,0 +1,16 @@
+import '@urql/core';
+
+declare module '@urql/core' {
+  interface DebugEventTypes {
+    /** Typescript won't let us explicitly state this value as undefined w/ no property */
+    teardown: undefined;
+    /** An execute[query|mutation|subscription] call. */
+    execution: {
+      sourceComponent: string;
+    };
+    /** An operation result with data. */
+    update: { value: any };
+    /** An operation result with error. */
+    error: { value: any };
+  }
+}
```

Comment:
I'd assume, this file may conflict with the "real" package, and I'm not sure if leaving it will cause issues with the repo wide `tsc` checks